Formula/allegro.rb

```diff
@@ -3,8 +3,8 @@ class Allegro < Formula
   homepage "http://liballeg.org/"
 
   stable do
-    url "http://download.gna.org/allegro/allegro/5.0.11/allegro-5.0.11.tar.gz"
-    sha256 "49fe14c9571463ba08db4ff778d1fbb15e49f9c33bdada3ac8599e04330ea531"
+  url "http://download.gna.org/allegro/allegro/5.2.0/allegro-5.2.0.tar.gz"
+  sha256 "af5a69cd423d05189e92952633f9c0dd0ff3a061d91fbce62fb32c4bd87f9fd7"
   end
```

Comment:
Please remove the `stable do` and `end` (we don't need a separate `stable do` block anymore) and move those two lines below `homepage` in addition to changing the indentation. Thanks!